### PR TITLE
Fix tiny SVG for board showcase

### LINKED ISSUE #141: Board SVGs are tiny now
E.g. 

image

### PR DESCRIPTION
**Pull request by AbdulrhmnGhanem**
closes #141 

### staging
image
### kitspace.org
image
### staging
image
### kitspace.org
image

### CODE REVIEW

File: frontend/src/components/Board/Readme.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ const Readme = ({ renderedReadme }) => (
         max-width: 100%;
       }
 
-      #readme .highlight {
+      #readme pre {
```
Comment (kasbah) [inline frontend/src/components/Board/Readme.js:25]: ?

Comment (AbdulrhmnGhanem) [reply frontend/src/components/Board/Readme.js:25]: I copied the `.highlight` from the original styles, this class doesn't exist in the readme rendered by Gitea.

Aside: this bug (the overflow) is reproducible on kitspace.org, see ulx3x board on a small screen.

**Comment by kasbah** [comment]
i deployed this to https://kaspar-dev.staging.kitspace.dev and it doesn't seem fixed (using firefox on deaktop and mobile at least).  also noticed a similar issue on mobile with the thumbnails.

**Comment by AbdulrhmnGhanem** [comment]
I just tried it on firefox, yes, it's broken. :confused:

**Comment by kasbah** [comment]
What's up with these tests after the merge @AbdulrhmnGhanem ?

**Comment by AbdulrhmnGhanem** [comment]
It seems unrelated but let's see if this is persistent.

**Comment by AbdulrhmnGhanem** [comment]
I rebased other PRs from master and the same test failure happened there, so it's a problem with the master, not this PR. I am investigating it now, what makes it wired is that no changes have been made to the upload functionality lately.

**Comment by kasbah** [comment]
Can you clean up this branch and merge please?

**Review by kasbah — APPROVED** [review]

**Pull request merged**